### PR TITLE
First pass at reorder of filters. Close #387.

### LINKED ISSUE #387: Update filters for Patients

### PR DESCRIPTION
**Pull request by vsmalladi**
Update patient fileter Close #387

### CODE REVIEW

File: src/encoded/schemas/patient.json [hunk 1]
```diff
@@ -141,137 +126,129 @@
...
+        },
+        "medical_imaging_before_nephrectomy.type" : {
+          "title": "Medical Imaging"
+        },
         "dominant_tumor.histology_filter": {
             "title": "Dominant Tumor"
         },
-        "dominant_tumor.stage": {
-            "title": "Dominant Tumor TNM Stage (at presentation)"
+        "surgery.pathology_report.laterality": {
```
Comment (mingjiecn) [inline src/encoded/schemas/patient.json]: should be "surgery.surgery_procedure.pathology_report.laterality"?

File: src/encoded/schemas/patient.json [hunk 2]
```diff
@@ -141,137 +126,129 @@
...
-            "title": "BMI"
-        },
-        "labs_and_vitals.BP_Systolic": {
-            "title": "BP_Systolic (mmHg)"
+        "metastasis.histology_proven": {
+            "title": "Metastasis Histology Proven"
         },
-        "labs_and_vitals.BP_Diastolic": {
-            "title": "BP_Diastolic (mmHg)"
+        "surgery.pathology_report.metasis_details.site": {
```
Comment (mingjiecn) [inline src/encoded/schemas/patient.json]: should be "surgery.surgery_procedure.pathology_report.metasis_details.site"?

File: src/encoded/schemas/patient.json [hunk 3]
```diff
@@ -141,137 +126,129 @@
...
+            "title":"Medication"
         },
-        "labs_and_vitals.Neutrophils": {
-            "title": "Neutrophils (10^3/ul)"
+        "medication_range":{
+            "title":"Medication Duration"
         },
-        "labs_and_vitals.Creatinine": {
-            "title": "Creatinine (mg/dL)"
+        "supportive_medication.name":{
```
Comment (mingjiecn) [inline src/encoded/schemas/patient.json]: supportive_medication name is not shown.

File: src/encoded/schemas/patient.json [hunk 4]
```diff
@@ -141,137 +126,129 @@
...
+            "title": "Radiation Treatment Status"
         },
-        "labs_and_vitals.Albumin": {
-            "title": "Albumin (g/dL)"
+        "radiation.site_consolidated" : {
+          "title": "Radiation Site"
         },
-        "labs_and_vitals.Sodium": {
-            "title": "Sodium (mmol/L)"
+        "radiation.dose_range": {
```
Comment (mingjiecn) [inline src/encoded/schemas/patient.json]: This one needs to be removed according to the issue #387

File: src/encoded/schemas/patient.json [hunk 5]
```diff
@@ -141,137 +126,129 @@
...
-        "biospecimen.anatomic_site_display": {
-            "title": "Specimen Anatomic Site"
-        },
         "biospecimen.species": {
             "title": "Specimen Species"
         },
-        "biospecimen.activity_status": {
-            "title": "Specimen Activity Status"
+        "biospecimen.biofile.assay_term_name": {
+            "title": "Genomics Assay"
```
Comment (mingjiecn) [inline src/encoded/schemas/patient.json:244]: This one should be Genomics Availability. Do we have this facet? May need to add a calculated field for it.

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Please check my single comments.

File: src/encoded/schemas/patient.json (hunk 5 above)
Comment (vsmalladi) [reply src/encoded/schemas/patient.json:244]: @mingjiecn  I think knowing the type of assay might be more helpful here than just a yes no. What do you think?

Comment (mingjiecn) [reply src/encoded/schemas/patient.json:244]: You are the expert here. So totally agree!

**Comment by SenseCode** [comment]
Remove Platelets in labs and vitals(no sure this one is extra field they don't need?)
Tumor Laterality is not showing
Supportive Medication is not showing
Others are fine. I think the genomic_release field is ok.

File: src/encoded/schemas/patient.json (hunk 5 above)
Comment (SenseCode) [reply src/encoded/schemas/patient.json:244]: This field we should show genomics data related to the patient, I think it's fine to show what kind of genomic assay type we have in database.

**Comment by vsmalladi** [comment]
Fixed laterality and supportive medication.

Left Platlets in as this was a typo on my part. Should have been there from the beging.

**Review by mingjiecn — APPROVED** [review]

**Review by SenseCode — APPROVED** [review]
All good to merge

**Pull request merged**